Documentation/kubernetes-on-baremetal.md

```diff
@@ -1,13 +1,34 @@
 # Kubernetes Installation on Bare Metal &amp; CoreOS
 
-This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS.
-After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
+This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS. After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
 
-## Deployment Prerequisites
+## Deployment Requirements
```

Comment:
I was not explicit, but intended it to be applied to all headings. See https://github.com/coreos/docs/blob/master/STYLE.md#headings

This has been contentious, but sentence case is both more modern and simpler to write without inciting debate about what word is of Title Consequence or Not. We still selectively employ Title Case in some visual design elements but the general rule for a general document is Sentence case.